### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Vegan-Food-Tel-Aviv** [s-vidal/react-week-1-s-vidal#1]
Hi Vidal,

Nice job! A few tips:

1. Instead defining styles in your js documents (i.e. style= { . . . }), create CSS classes in your .css file(s) and apply the custom classes in your JSX code like how we did in HTML/JS. 

2. Any time you define a new class, put it in its own component/file. Accordingly, each file should generally have only one class declaration. You do a good job of this generally, I just noticed at least one time when you didn't.

**Comment by JonathanGrossmanITC** [comment]
Hey, the message was from me! I was in the wrong GitHub account.

**Comment by s-vidal** [comment]
Yes I was wondering what that vegan thing was.Ok it makes sense, thanks for
the feedback!
